CONTRIBUTING.md

```diff
@@ -126,6 +126,11 @@ Examples:
   - For a new translation of an existing page: `cp: add Tamil translation`
   - For related changes to several pages: `grep, find, locate: synchronize format of wildcards`
 
+## Inclusive language
+
+Where possible, use inclusive language in the content of pages. For example, prefer terms like "denylist"/"allowlist" instead of "blacklist"/"whitelist", "primary"/"secondary" instead of "master"/"slave", "one" instead of "him"/"her", etc.
```

Comment:
Good call with "one", I didn't think of that. However, I'd say [singular they](https://en.wikipedia.org/wiki/Singular_they) is more versatile as a neutral pronoun. Revisiting this passage, maybe I'd suggest a slight tweak to make this more evident:

> "they"/"them"/"their" instead of "he"/"him"/"his"

WDYT?

Reply:
That's a good point, @waldyrious - I think I agree about singular they.

Reply:
I disagree - it's grammatically incorrect. I prefer the use of "one".

Reply:
I'm not sure what controversy you have in mind regarding the use of singular they — I'm not aware of any. In any case, assuming they're roughly equivalent in terms of desirability, and since we all agree that "one" is more formal, I'd personally go with "they" instead, so as to encourage a more informal and warm tone in our community.

Reply:
"Singular they" is a relatively new concept and not accepted as grammatically correct by everyone. I changed the docs to they, regardless.

Reply:
It's not actually as new as you think - in the Wikipedia article that @waldyrious links to, it gives examples dating back to 1375.

Reply:
Regardless of the merits of each approach, since we've reached agreement to use "they", I'll mark this conversation as resolved.